nilearn/interfaces/fmriprep/load_confounds_scrub.py

```diff
@@ -87,6 +89,15 @@ def extract_outlier_regressors(confounds):
         outliers = pd.DataFrame()
     confounds = confounds.loc[:, confounds_cols]
     sample_mask = _outlier_to_sample_mask(outliers)
+
+    if sample_mask is not None and sample_mask.size == 0:
+        warnings.warn(
+            category=RuntimeWarning,
+            message="All time points in the confounds were marked as "
+            "outliers. This would lead to all volumes in the time "
+            "series to be scrubbed.",
+            stacklevel=2,
```

Comment:
```suggestion
            stacklevel=4,
```

Given that extract_outlier_regressors is not part of the public API, users are unlikely to use it directly. I suspect we can raise the stacklevel higher: from a quick check in the codebase I think 4 should do it.